README.md

```diff
@@ -233,8 +233,8 @@ mark checks as ignored.
 
 For example, `GH101` expects all your action files to have a nice `name:` field.
 If you are happy with the file-based names you see in CI, you should feel free
-to simply ignore this check (just visually ignore it for the moment, a way to
-specify ignored checks will likely be added eventually).
+to simply ignore this check (you can specify ignored checks in pyproject.toml or
+by passing args to repo-review, see the [repo-review docs][repo-review]).
```

Comment:
I think I would also link https://repo-review.readthedocs.io/en/latest/intro.html#configuring and/or https://repo-review.readthedocs.io/en/latest/cli.html